src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -145,7 +144,11 @@ public Response createItem(Item payload) throws Exception {
     String contactEmail = payload.getContactEmail();
     String contactPhone = payload.getContactPhone();
     String termsOfDelivery = payload.getTermsOfDelivery();
-    
+    Boolean allowDelivery = payload.isAllowDelivery() != null ? payload.isAllowDelivery() : false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the literal "false" boolean value. ([squid:S1125](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1125))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=39&issues=AWXYx1xjlgbYnohYXER4&open=AWXYx1xjlgbYnohYXER4)

[ISSUE_KEY]: AWXYx1xjlgbYnohYXER4